src/App.tsx

```diff
@@ -17,18 +88,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                handleQueryChange={handleQueryChange}
+                query={query}
+                handleQueryDelete={handleQueryDelete}
+                handleFilterCompleted={handleFilterCompleted}
+                handleFilterActive={handleFilterActive}
+                handleFilterAll={handleFilterAll}
```

Comment:
replace it with one function handleFilterChange which accepts new filter value as an argument

Reply:
Not fixed